compiler/luci-interpreter/src/loader/GraphLoader.cpp

```diff
@@ -87,6 +87,7 @@ bool isExecutableNode(const luci::CircleNode *node)
     case luci::CircleOpcode::CIRCLESPLITOUT:
     case luci::CircleOpcode::CIRCLESPLITVOUT:
     case luci::CircleOpcode::CIRCLEUNPACKOUT:
+    case luci::CircleOpcode::CIRCLEVARIABLE:
     case luci::CircleOpcode::CIRCLEWHILEOUT:
```

Comment:
Could you please add your changes from this file, after that I will add changes to luci-interpreter

Reply:
#8329 posted